autoware_launch/CHANGELOG.rst

```diff
@@ -0,0 +1,2315 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package autoware_launch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.38.0 (2024-11-13)
+-------------------
+* feat(start/goal_planner): increse max dry steering angle (`#1200 <https://github.com/youtalk/autoware_launch/issues/1200>`_)
```

Comment:
Could you replace the links to issues from `youtalk` -> `autowarefoundation` ?

(Esteve already fixed it in Autoware.Universe: https://github.com/autowarefoundation/autoware.universe/pull/9304/files)

Reply:
Good catch! I fixed them. https://github.com/autowarefoundation/autoware_launch/pull/1226/commits/5078d8c15e92675cc16e1fe4619c2981ca234163
From now on, instead of creating branch PRs from my forked repository, I will create branch PRs directly from the autowarefoundation repository.